awx/ui/package.json

```diff
@@ -136,6 +136,7 @@
     "rrule": "git+https://git@github.com/jkbrzt/rrule#4ff63b2f8524fd6d5ba6e80db770953b5cd08a0c",
     "select2": "^4.0.2",
     "sprintf-js": "^1.0.3",
+    "timezone-js": "github:ansible/timezone-js#0.4.14",
```

Comment:
We actually needed this as a top-level dependency, but I don't think it was explicitly added. I _think_ the reason it didn't matter before is that I our `npm install` command was falling back to the `package.json` or some other nuanced behavior that the `ci` command won't do.